content/blog/2020-MM-DD-treedata.table/index.md

```diff
@@ -0,0 +1,82 @@
+There are other ways to achieve similar results, but as seen on Fig. 1, these are not as efficient.
+
+Something unique in `treedata.table` is the ability to work with `multiPhylo` objects.
+`multiPhylo` objects, as their name suggests, are colelctions of trees.
+For example, perhaps you would like to match all the trees in a Bayesian posterior sample to a dataset to perform
+a comparative methods analysis for every tree in your sample.
+Our second [vignette](https://ropensci.github.io/treedata.table/articles/B_multiphylo_treedata.table.html) walks researchers through the process of syncing many trees to one dataset, manipulating the data, and performing a continuous trait evolution modeling exercise in [geiger](https://cran.r-project.org/web/packages/geiger/index.html).   
+
+Our other two vignettes cover other important data functions researchers may wish to do with phylogenetic data.
+For example, syncing names across datasets and phylogenetic trees based on [partial matching](https://ropensci.github.io/treedata.table/articles/C_PartialMatching.html), and [assorted data tidying functions](https://ropensci.github.io/treedata.table/articles/D_AdditionalFunctions_treedata.table.html).
+
+### Where did `treedata.table` come from?
+<div style="text-align:center">
+<img src='img/NantucketBeach.jpg' align="middle"width="600" />
+</div>
+<font size="2"> **Fig. 2.** The beach outside the workshop bunkhouse. </font>
+
+`treedata.table` is a research output from the [Nantucket DevelopR workshop](https://github.com/NantucketDevelopeR/2019Workshop).
+Funded by an NSF grant to [Liam Revell](http://www.faculty.umb.edu/liam.revell/), the workshop aimed to support intermediate R developers working with phylogenetic and phylogenetic comparative methods.
+Last time around, we had four instructors ([April Wright](paleantology.com), [Josef Uyeda](http://www.uyedalab.com/), [Claudia Solis-Lemus](https://solislemuslab.github.io/) and [Klaus Schliep](https://www.phangorn.org/)).
+Each instructor taught one serious lesson during the week, and most did one "fireside chat" about being a computational scientist working in R.
+
+But the real heart of the workshop was spending the day immersed in coding, both collaboratively and alone.
+Participants each came with a project in mind, and made pitches to others for their project.
+Some participants worked on multiple projects, or one with a group and one alone.
+The instructors were housed separately, and often we'd show up to the beachside house to find everyone didn't want lecture yet - they were too busy working already.
+Intermediate programmers are often sort of left to their own devices in education, and spending a week developing skills, community, and friendships was absolutely lovely. And, as evidenced by this package, productive!
+
+### Acknowledgements
+
+We'd like to thank all the participants at the workshop for a very special week together in an amazing place.
```

Comment:
Could you add an acknowledgement of the peer-reviewers of treedata-table as well as a link to the peer-review issue?